Source/diablo.cpp

```diff
@@ -103,6 +103,7 @@ static void print_help_and_exit()
 	printInConsole("    %-20s %-30s\n", "--data-dir", "Specify the folder of diabdat.mpq");
 	printInConsole("    %-20s %-30s\n", "--save-dir", "Specify the folder of save files");
 	printInConsole("    %-20s %-30s\n", "--config-dir", "Specify the location of diablo.ini");
+	printInConsole("    %-20s %-30s\n", "--ttf-file", "Specify the location of CharisSLIB.ttf");

```

Comment:
Best not to put the specific font tile name here. I can be different depending on builds.

Reply:
Oh I thought that was the only font we used.

Reply:
Some systems specify a different font to use less memory.

Reply:
+ typo in name too

Reply:
I would prefer `ttf-dir`. TTF file can still be a separate setting.

If we swap out the font in the future, we don't want the distributions to have change the flag.